Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG PYTORCH_IMAGE=nvcr.io/nvidia/pytorch:19.10-py3
```

Comment:
NGC dockers suppose to run in `nvidia-docker` and `--runtime=nvidia`:
https://docs.nvidia.com/deeplearning/frameworks/user-guide/index.html#runcont
So I think maybe you need to build docker and highlight that to make sure GPUs are working fine.